src/pages/Lend/LoanChannelCategoryExperiment.vue

```diff
@@ -86,117 +152,822 @@ const pageQuery = gql`
 	}
 `;
 
+const categoryProperties = {
+	women: {
+		title: 'Women',
+		tagline: 'Support women across a variety of causes',
+		image: imageRequire('./lbc-women.jpg'),
+		loanChannelId: 5,
+		subCategories: [
+			{
+				name: 'Entrepreneurs',
+				variables: {
+					filters: {
+						distributionModel: 'both',
+						gender: 'female',
+						sector: [7, 4],
+						status: 'fundraising'
+					},
+					sortBy: 'popularity'
+				}
+			},
+			{
+				name: 'Single moms',
+				variables: {
+					filters: {
+						distributionModel: 'both',
+						gender: 'female',
+						loanTags: [17],
+						status: 'fundraising'
+					},
+					sortBy: 'popularity'
+				}
+			},
+			{
```

Comment:
It's concerning that we are swapping and or dropping regions in these selections. Can we please roll Oceania into this somewhere, specifically Fiji and Timor-Leste current have about 90 loans.